content/kbld/_index.html

```diff
@@ -0,0 +1,241 @@
+                </div>
+                <div class="content">
+                    <h3>Delegates building to proven tools</h3>
+                    <p>Orchestrates image builds (delegates to tools like Docker) and registry pushes.</p>
+                </div>
+            </div>
+            <div class="col">
+                <div class="icon">
+                    <img src="/img/overlays.svg" />
+                </div>
+                <div class="content">
+                    <h3>Saves sources of built images</h3>
+                    <p>Records metadata about image sources in annotation on Kubernetes resources <a href="#examples">(see examples below).</a></p>
+                </div>
+            </div>
+            <div class="col">
+                <div class="icon">
+                    <img src="/img/yaml.svg" />
+                </div>
+                <div class="content">
+                    <h3>Immutable image references</h3>
+                    <p>Resolves image references to their digest form (immutable) (<a href="#examples">see "why" below</a>).</p>
+                </div>
+            </div>
+            <div class="col">
+                <div class="icon">
+                    <img src="/img/yaml.svg" />
+                </div>
+                <div class="content">
+                    <h3>Air-gap environments</h3>
+                    <p>Provides a way to transport set of images in a single tarball between registries.</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="code-block" id="examples">
+        <h2>Basic Usage</h2>
+        <div class="code">
+            <p class="header">Basic Usage</p>
+            <p class="comment"># Configurations picked up from a directory</p>
+            <p>$ <span class="green-text">kbld -f examples/cassandra/ | kubectl apply -f -</span></p>
+            <p class="comment"># Can be used with helm charts</p>
+            <p>$ <span class="green-text">helm template my-chart --values my-vals.yml | kbld -f - | kubectl apply -f -</span></p>
+            <p class="comment"># ... and with kustomize</p>
+            <p>$ <span class="green-text">kustomize build ./some-app | kbld -f - | kubectl apply -f -</span></p>
+            <p class="comment"># ... or templated with ytt and deployed with kapp</p>
+            <p>$ <span class="green-text">ytt -f ./some-app | kbld -f - | kapp -y deploy -a some-app -f -</span></p>
+        </div>
+    </div>
+    <div class="code-block">
+        <h3>Examples</h3>
+        <p>Resolves name-tag pair reference (<span class="example">nginx:1.17</span>) into digest reference (<span class="example">index.docker.io/library/nginx@sha256:2539d4344...</span>)</p>
+        <p>Input:</p>
+        <div class="code">
+            <p>apiVersion: apps/v1</p>
+            <p>kind: Deployment</p>
+            <p>metadata:</p>
+            <p><span class="tab-space"></span>name: nginx-deployment</p>
```

Comment:
seems like code content like this is best to be done via pre. is there a reason why it's using individual P-s for each line?